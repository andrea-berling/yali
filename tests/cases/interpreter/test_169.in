class world {
  inworld() {
    print "from world";
  }
}
class hello < world {
  inhello() {
    print "from hello";
  }
}
class foo < hello {
  infoo() {
    print "from foo";
  }
}
// foo should inherit the methods
// from both world and hello
var foo = foo();
foo.inworld();
foo.inhello();
foo.infoo();

