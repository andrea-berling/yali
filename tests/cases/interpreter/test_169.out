from world
from hello
from foo
